python/tskit/tables.py

```diff
@@ -2213,6 +2227,23 @@ def compute_mutation_parents(self):
         self.ll_tables.compute_mutation_parents()
         # TODO add provenance
 
+    def compute_mutation_times(self):
+        """
+        Modifies the tables in place, computing valid values for the ``time`` column of
+        the mutation table. For this to work, the node and edge tables must be
+        valid, and the site and mutation tables must be sorted (see
+        :meth:`TableCollection.sort`).
```

Comment:
And indexed

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/672/commits/9d255cd75c2be687535eb6333f64a09e4aa597ce